constants.js

```diff
@@ -0,0 +1,7 @@
+const QOS_LEVELS = {
+    AT_MOST_ONCE: 0,
+    AT_LEAST_ONCE: 1,
+    EXACTLY_ONCE: 2
+};
+
+module.exports = { QOS_LEVELS };
```

Comment:
Will it make sense to export this constant from index.js? To allow use "named values" (not 0, 1, 2) outside the library?